test-requirements.txt

```diff
@@ -0,0 +1,6 @@
+odoo-addon-document_knowledge @ git+https://github.com/OCA/knowledge.git@refs/pull/503/head#subdirectory=document_knowledge
+odoo-addon-document_page @ git+https://github.com/OCA/knowledge.git@refs/pull/504/head#subdirectory=document_page
+odoo-addon-document_page_procedure @ git+https://github.com/OCA/management-system.git@refs/pull/610/head#subdirectory=document_page_procedure
+odoo-addon-mgmtsystem_action @ git+https://github.com/OCA/management-system.git@refs/pull/605/head#subdirectory=mgmtsystem_action
+odoo-addon-mgmtsystem_survey @ git+https://github.com/OCA/management-system.git@refs/pull/609/head#subdirectory=mgmtsystem_survey
+odoo-addon-mgmtsystem_nonconformity @ git+https://github.com/OCA/management-system.git@refs/pull/611/head#subdirectory=mgmtsystem_nonconformity
```

Comment:
surely all of them needed for this PR? I see only 2 links in PR description.

Reply:
I missed adding some details to the PR description. I’ve now updated the test-requirements.txt and corrected the PR description.